packages/data-point/lib/entity-types/entity-request/resolve.js

```diff
@@ -104,23 +105,31 @@ function resolveRequest (acc, resolveReducer) {
   return rp(acc.options)
     .then(result => utils.set(acc, 'value', result))
     .catch(error => {
+      // remove auth objects from acc and error for printing to console
+      const redactedAcc = fp.set('options.auth')('[omitted]')(acc)
+      const redactedError = fp.set('options.auth')('[omitted]')(error)
+
       const message = [
         'Entity info:',
         '\n  - Id: ',
-        _.get(acc, 'reducer.spec.id'),
+        _.get(redactedAcc, 'reducer.spec.id'),
         '\n',
-        utils.inspectProperties(acc, ['options', 'params', 'value'], '  '),
+        utils.inspectProperties(
+          redactedAcc,
+          ['options', 'params', 'value'],
+          '  '
+        ),
         '\n  Request:\n',
         utils.inspectProperties(
-          error,
+          redactedError,
           ['error', 'message', 'statusCode', 'options', 'body'],
           '  '
         )
       ].join('')
 
       // this is useful in the case the error itself is not logged by the
       // implementation
-      console.info(error.toString(), message)
+      console.info(redactedError.toString(), message)
```

Comment:
The redacted versions are for printing, but the original `error` is thrown. `error.message` is the redacted message, so even if that is snapshot it's clear of creds.